src/main/java/com/netflix/blitz4j/LoggingConfiguration.java

```diff
@@ -162,22 +162,22 @@ public void configure(Properties props) {
                 URL url = Loader.getResource(LOG4J_PROPERTIES);
                 if (url != null) {
                     try (InputStream in = url.openStream()) {
-                        this.props.load(in);
+                        this.initialProps.load(in);
                     }
                 }
             } catch (Exception t) {
                 System.err.println("Error loading properties from " + LOG4J_PROPERTIES);
             } 
+        }   
+        
+        Enumeration enumeration = props.propertyNames();
+        while (enumeration.hasMoreElements()) {
+            String key = (String) enumeration.nextElement();
+            String propertyValue = props.getProperty(key);
+            this.initialProps.setProperty(key, propertyValue);
         }
-        if (props != null) {
```

Comment:
Props should never be null.  I do not to make sure of that at the top of this constructor.